packages/vite/client.d.ts

```diff
@@ -246,3 +246,20 @@ declare module '*?inline' {
   const src: string
   export default src
 }
+
+declare interface VitePreloadErrorEvent extends Error {
```

Comment:
```suggestion
declare interface VitePreloadErrorEvent extends Event {
```

Should this extend from `Event`?

Reply:
Yeah, this seems more reasonable.